.circleci/config.yml

```diff
@@ -646,21 +655,47 @@ jobs:
             - run:
                 name: Ensure package.json Version And Tag Match
                 command: cd desktop/bin && node validate_tag.js $VERSION
-      - run: *update-node
-      - run: *yarn-install
+      - run:
+          name: Install Node
+          command: |
+            set +e
+            set +x
+            export NVM_DIR="/opt/circleci/.nvm"
+            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+            nvm install
+            nvm use
+      - run:
+          name: Install Dependencies
+          command: |
+            export NVM_DIR="/opt/circleci/.nvm"
+            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+            nvm use
+            npm install -g yarn
+
+            yarn config set chromedriver_skip_download true
```

Comment:
Necessary to silence a chromedriver resolution/installation error. In contrast to the main Calypso job, for some reason the `DETECT_CHROMEDRIVER_VERSION` and `CHROMEDRIVER_FILEPATH` environment variables don't work here.

Reply:
Out of curiosity, what does it mean "don't work"? As in, they are ignored?

You could also add them to the `environment:` section in line 32.

Reply:
Yeah - strangely enough declaring them in the `environment` section didn't seem to make a difference - not sure why. I'll take another stab at it and a few other refactors in a downstream PR.